### ISSUE TITLE
didn't work

### ISSUE THREAD

**Issue by ghost** [ammulder/AppleIntelWiFiMVM#25]
我在3165，10.12.6上实验了，完全无法识别。
I have just tested it with 3165 wireless card on sierra10.12.6 and it just didn't work.

**Comment by ghost** [comment]
It won't

"This driver is still in the "can it be done?" stage.

As of this writing the driver recognizes the appropriate hardware and loads a matching firmware file, but does not yet send the firmware to the hardware.

Now I'm in the process of trying to get all the Linux driver code to build on Mac OS X. This will be a very long process. Work is currently happening on the "Porting" branch, but only a small subset of the code even compiles right now.

Long story short, this driver does not yet actually connect to wireless networks.

If you're interested in helping, it would be great to have more people try test builds just to ensure it accurately identifies all the hardware I don't have on hand to try."

That's what the developer put in the README.MD
So just be patient.

**Comment by ghost** [comment]
I have a card but now I changed it to BCM94352z to surface internet. thanks for your invention. Making intel wireless get work in macOS is my dream, too. If you need some help ,just tell me .

**Comment by ghost** [comment]
Yes this project definitely needs help, noone else seems to care. Please contribute - fork and start coding
